zbus/src/lib.rs

```diff
@@ -913,7 +913,7 @@ mod tests {
             "org.zbus.Issue260",
         )
         .await?
-        .call("Whatever", &())
+        .call::<_, _, ()>("Whatever", &())
```

Comment:
It's best to keep the `clippy::` prefix in the subject or mentioned in the description of the commit so the context is clear. I generally tend to add the clipping warning/error message to the description even so it's clear from the commit itself, what the change is all about.

Reply:
hmm.. actually I can't find this lint [in clippy](https://rust-lang.github.io/rust-clippy/master/index.html#/). It's not clippy? This would be why it's good to add details to the commit.

Reply:
ok, added compiler warning

Reply:
Cool but in the future, please mark it as the verbatim output, just like you did for the clippy warning in c1789b6cbdbef1fc59d750e1ee262055c97b81ca.